src/components/Menu/index.tsx

```diff
@@ -127,6 +127,11 @@ const MenuItem = styled(ExternalLink)`
     cursor: pointer;
     text-decoration: none;
   }
+  :focus-visible {
```

Comment:
Should this be on ExternalLink, so it correctly displays for all external links?

Reply:
https://github.com/Uniswap/interface/pull/3070#discussion_r780028757